app/invocation/invocation_action_card.tsx

```diff
@@ -77,20 +90,15 @@ export default class InvocationActionCardComponent extends React.Component<Props
       action.commandDigest.sizeBytes;
     rpcService
       .fetchBytestreamFile(commandFile, this.props.model.getId(), "arraybuffer")
-      .then((action_buff: any) => {
-        let temp_array = new Uint8Array(action_buff);
+      .then((actionBuff: any) => {
+        let tempArray = new Uint8Array(actionBuff);
         this.setState({
           ...this.state,
```

Comment:
nit: no need to spread `...state`, and also would be cleaner without the `tempArray` variable